ui/src/main/java/com/hyperwallet/android/ui/transfermethod/ListTransferMethodFragment.java

```diff
@@ -373,16 +350,20 @@ void replaceData(List<HyperwalletTransferMethod> transferMethodList) {
 
 
             void bind(@NonNull final HyperwalletTransferMethod transferMethod) {
+                String type = transferMethod.getField(TYPE);
+                final String transferMethodIdentification = TransferMethodUtils.getTransferMethodDetail(
```

Comment:
please use import static as it's done in getStringResourceByName and getStringFontIcon